metrics/network/iperf3_kubernetes/k8s-network-metrics-iperf3.sh

```diff
@@ -116,6 +120,31 @@ EOF
 	fi
 }
 
+function iperf3_parallel() {
```

Comment:
just for better understanding to the user, should we add a comment that the test will use 4 parallel connections?